common-npm-packages/azure-arm-rest-v2/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "azure-pipelines-tasks-azure-arm-rest-v2",
-  "version": "3.219.0",
+  "version": "3.219.1",
```

Comment:
Normally, version should be 220 but since we're trying to keep in sync with other package we have used 219.